src/components/bookForm.js

```diff
@@ -1,14 +1,36 @@
-import React from 'react';
-
-const BookForm = () => (
-  <>
-    <h3 style={{ marginTop: '2rem' }}>ADD NEW BOOK</h3>
-    <form style={{ display: 'flex', gap: '2rem' }}>
-      <input type="text" placeholder="Title" />
-      <input type="text" placeholder="Author" />
-      <button type="button">Add Book</button>
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/bookSlice';
+
+const Form = () => {
+  const [book, setBook] = useState({ booktitle: '', author: '' });
```

Comment:
- [ ] As indicated in the previous review, kindly ensure that the Book form does not accept empty values. An workaround this may be to consider wrapping this around _conditionals_.

**Empty values added:**
![image](https://user-images.githubusercontent.com/12745474/223975875-a60f203f-cc3e-4edf-88ca-3ebf568cfedc.png)